.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 # This file is managed by ModuleSync.
 
 * @opus-codium/core
+* @opus-codium/vittoria-conseil
```

Comment:
@smortex That's great but could you replace vittoria-conseil with vittoria-io

Reply:
We'll do this next: we have a few repos already using this name and this is already half committed (in #64).  When this PR gets merged, we will be able to update this toil without bothering the rest of @opus-codium/core :wink:.